Sources/TuistKit/Services/GraphService.swift

```diff
@@ -98,3 +116,39 @@ final class GraphService {
         try System.shared.runAndPrint(["brew", "install", "graphviz"], verbose: false, environment: env)
     }
 }
+
+private enum GraphServiceError: FatalError {
+    case invalidFormat(String)
+    case encodingError(String)
+    
+    var description: String {
+        switch self {
+        case .invalidFormat(let format):
+            return "\(format) is not valid for export"
+        case .encodingError(let format):
+            return "failed to encode graph to \(format)"
+        }
+    }
+    
+    var type: ErrorType {
+        switch self {
+        case .invalidFormat:
+            return .abort
```

Comment:
I think this should be a `.bug` since it should never happen, right?

Reply:
Yeah. sounds good.